CMakeLists.txt

```diff
@@ -267,6 +267,13 @@ check_include_file(siginfo.h HAVE_SIGINFO_H)
 check_include_file(endian.h HAVE_ENDIAN_H)
 
 # Find libraries
+find_package(PkgConfig REQUIRED) # needed to find various libraries
```

Comment:
Could you help me understand why this was added? There is no corresponding removal of `find_package(PkgConfig REQUIRED)` from another location.

Reply:
hmm  This was just part of me moving find_package() to root.

I should pull that out to a different PR